tests/migrations/test_migrations.py

```diff
@@ -381,3 +381,7 @@ def test_migration_to_5492f53d1fa5(alembic_runner: MigrationContext, alembic_eng
 
     assert "validation_results" in [c["name"] for c in inspector.get_columns("submission")]
     assert "validation_json" not in [c["name"] for c in inspector.get_columns("submission")]
+
+
+def test_migration_to_c7238487f08d(alembic_runner: MigrationContext, alembic_engine: Engine):
+    alembic_runner.migrate_up_to("c7238487f08d")
```

Comment:
We don't need to add these anymore.  The import:
```
from pytest_alembic.tests import (
    test_single_head_revision,  # noqa: F401
    test_up_down_consistency,  # noqa: F401
    test_upgrade,  # noqa: F401
)
```
takes care of testing all upgrade/downgrades